tools/StaticAnalysis/Exceptions/AzureRM.Compute/BreakingChangeIssues.csv

```diff
@@ -21,3 +21,8 @@
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.RemoveAzureRmContainerService","Remove-AzureRmContainerService","0","1020","The cmdlet 'Remove-AzureRmContainerService' no longer has output type 'System.Void'.","Make cmdlet 'Remove-AzureRmContainerService' return type 'System.Void'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.RemoveAzureRmVmssDiagnosticsExtension","Remove-AzureRmVmssDiagnosticsExtension","0","3010","The property 'IdentityIds' of type 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetIdentity' has been removed.","Add the property 'IdentityIds' back to type 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetIdentity'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.NewAzureAdditionalUnattendContentCommand","Add-AzureRmVMAdditionalUnattendContent","0","3010","The property 'IdentityIds' of type 'Microsoft.Azure.Management.Compute.Models.VirtualMachineIdentity' has been removed.","Add the property 'IdentityIds' back to type 'Microsoft.Azure.Management.Compute.Models.VirtualMachineIdentity'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.AddAzureVMDataDiskCommand","Add-AzureRmVMDataDisk","0","3000","The type of property 'InstanceView' of type 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSetVM' has changed from 'Microsoft.Azure.Management.Compute.Models.VirtualMachineInstanceView' to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetVMInstanceView'.","Change the type of property 'InstanceView' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineInstanceView'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.AddAzureVMDataDiskCommand","Add-AzureRmVMDataDisk","0","3000","The type of property 'InstanceView' of type 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSetVM' has changed from 'Microsoft.Azure.Management.Compute.Models.VirtualMachineInstanceView' to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetVMInstanceView'.","Change the type of property 'InstanceView' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineInstanceView'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.GetAzureRmVmssVM","Get-AzureRmVmssVM","0","3000","The type of property 'InstanceView' of type 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSetVM' has changed from 'Microsoft.Azure.Management.Compute.Models.VirtualMachineInstanceView' to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetVMInstanceView'.","Change the type of property 'InstanceView' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineInstanceView'."
```

Comment:
These two types do not contain the same set of properties. So, these 3 above are also breaking changes. You need a PowerShell model that matches the same properties of the currently existing model.